README.md

```diff
@@ -215,6 +215,24 @@ unsafe fn
 ...
 ```
 
+## FAQ
+
+* **Does this library support data types other than `str` and `[u8]`?
+  (e.g., structures implementing `Eq`.)**
+
+  Not supported. This library uses Aho-Corasick automata built with a
+  data structure called *double-array trie*. The algorithm on this data
+  structure works with XOR operations on the input haystack. Therefore,
+  the haystack must be a sequence of integers.
```

Comment:
I don't think this is an answer about not supporting `[u32]` and `[u64]`. In principle, DAAC can support those types. In my understanding,

- the reason for supporting `[u8]` is that `bytewise::daachorse` is optimized for byte sequences based on the compact double-array, and
- the reason for supporting `str` is that `charwise::daachorse` is optimized for (Asian) natural language texts based on the code mapping technique.

We could explain them by citing the corresponding sections of our arXiv paper.

Reply:
The first version only describes why daachorse does not support non-integer sequences, so I added the following sentence at the end of paragraph:
>  This library is specially optimized for `str` and `[u8]` among integer sequences.